### PR TITLE
Improve Fruit Distribution + RFPManageForm

### PR DESCRIPTION
**Pull request by qusvv**
Issue #19 : improve fruit distribution form:

- added a collapsible form to add a new recipient
- only owner + pickers + pick-leader + beneficiary organizations are listed in the recipient dropdown
- only trees selected for the harvest are listed in the tree dropdown
- added a delete button with confirmation

The only drawback that i see is that when adding or deleting a distribution page reloads on top and we still need to scroll back down but I think it is a way less tedious process than before.

Warning: along the way I have added the `django-debug-toolbar` which i find very useful to check which static files have been loaded, among other things: to install it: `$(venv) pip3 install django-debug-toolbar`

### CODE REVIEW

File: saskatoon/saskatoon/settings.py [hunk 1]
```diff
@@ -77,6 +80,7 @@
     'rest_framework',
     'django_filters',
     'crispy_forms',
+    'debug_toolbar',
```
Comment (tristanlatr) [inline saskatoon/saskatoon/settings.py:83]: Maybe the debug toolbar could be enabled only if `SASKATOON_DEBUG` is True ?

File: saskatoon/sitebase/static/js/confirm/delete.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+function confirmDelete() {
+    if (confirm('Are you sure you want to delete this entry?')) {
+        // console.log("Entry Deleted");
```
Comment (tristanlatr) [inline saskatoon/sitebase/static/js/confirm/delete.js:4]: Should it return True ?

**Review by tristanlatr — APPROVED** [review]
It looks great! 
I haven't tested it yet though.

File: saskatoon/sitebase/static/js/confirm/delete.js (hunk 2 above)
Comment (qusvv) [reply saskatoon/sitebase/static/js/confirm/delete.js:4]: No when adding `return true;` the entry will get deleted when you click cancel. I think you just want it to pass. 
I also tried:
```
function confirmDelete() {
      return confirm('Are you sure you want to delete this entry?')
}
```
but didn't work

File: saskatoon/saskatoon/settings.py (hunk 1 above)
Comment (qusvv) [reply saskatoon/saskatoon/settings.py:83]: I'm pretty certain django does that by default

Comment (tristanlatr) [reply saskatoon/saskatoon/settings.py:83]: ok1

File: saskatoon/sitebase/static/js/confirm/delete.js (hunk 2 above)
Comment (tristanlatr) [reply saskatoon/sitebase/static/js/confirm/delete.js:4]: Ok!

**Pull request merged**